msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AadInstanceDiscoveryProvider.java

```diff
@@ -80,7 +79,16 @@ static InstanceDiscoveryMetadataEntry getMetadataEntry(URL authorityUrl,
         InstanceDiscoveryMetadataEntry result = cache.get(host);
 
         if (result == null) {
-            doInstanceDiscoveryAndCache(authorityUrl, validateAuthority, msalRequest, serviceBundle);
+            if(msalRequest.application().instanceDiscovery()){
+                doInstanceDiscoveryAndCache(authorityUrl, validateAuthority, msalRequest, serviceBundle);
+            } else {
+                // instanceDiscovery flag is set to False. Do not perform instanceDiscovery.
```

Comment:
Very minor, but missing a 't': 'Do no**t** perform instanceDiscovery'

Reply:
Do you not see 't'. I see it :D

Reply:
Oops, I meant to put this comment in the other PR. The 't' is missing from the word 'not' in that version: https://github.com/AzureAD/microsoft-authentication-library-for-java/pull/570/files#r1035238440